app/grants/views.py

```diff
@@ -538,6 +538,9 @@ def get_grants(request):
     page = request.GET.get('page', 1)
     collections_page = request.GET.get('collections_page', 1)
     sort = request.GET.get('sort_option', 'weighted_shuffle')
+    if request.user.is_authenticated and request.user.profile.pk % 2 == 1:
+            if sort == 'weighted_shuffle':
+                sort = 'random_shuffle'
```

Comment:
The indentation here looks off.

Reply:
Yup it def is off ! @owocki you sure this worked locally ?  I'm fixing it up for now cause i need everything in round 8 branch 

nice catch @aamustapha